<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangTidy::ProjectConfigPage</class>
 <widget class="QWidget" name="ClangTidy::ProjectConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>379</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="kcfg_useConfigFile">
     <property name="text">
      <string>&amp;Use .clang-tidy file(s)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="checksTab">
      <attribute name="title">
       <string>Checks</string>
      </attribute>
      <property name="enabled">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ClangTidy::CheckSetSelectionComboBox" name="kcfg_checkSetSelection">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ClangTidy::CheckSelection" name="enabledChecks" native="true"/>
       </item>
       <item>
        <widget class="ClangTidy::CustomCheckSetConfigProxyWidget" name="kcfg_enabledChecks" native="true">
         <property name="visible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Includes</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Check system headers:</string>
         </property>
         <property name="buddy">
          <cstring>kcfg_checkSystemHeaders</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="kcfg_checkSystemHeaders"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="headerFilterLabel">
         <property name="text">
          <string>Header fi&amp;lter:</string>
         </property>
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="buddy">
          <cstring>kcfg_headerFilter</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="kcfg_headerFilter">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Regular expression matching the names of the headers to output diagnostics from. Diagnostics from the main file of each translation unit are always displayed. Can be used together with -line-filter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Extra Parameters</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>E&amp;xtra parameters:</string>
         </property>
         <property name="buddy">
          <cstring>kcfg_additionalParameters</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="kcfg_additionalParameters">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Additional command line options to pass to clang-tidy.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClangTidy::CheckSetSelectionComboBox</class>
   <extends>QComboBox</extends>
   <header>config/checksetselectioncombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ClangTidy::CheckSelection</class>
   <extends>QWidget</extends>
   <header>config/checkselection.h</header>
  </customwidget>
  <customwidget>
   <class>ClangTidy::CustomCheckSetConfigProxyWidget</class>
   <extends>QWidget</extends>
   <header>config/customchecksetconfigproxywidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_useConfigFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>checksTab</receiver>
   <slot>setDisabled(bool)</slot>
  </connection>
  <connection>
   <sender>kcfg_useConfigFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_headerFilter</receiver>
   <slot>setDisabled(bool)</slot>
  </connection>
  <connection>
   <sender>kcfg_useConfigFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>headerFilterLabel</receiver>
   <slot>setDisabled(bool)</slot>
  </connection>
 </connections>
</ui>
